preprocessing.py

```diff
@@ -1 +1,34 @@
 #This is for preprocessing: First will be relabeling everything and then will be splitting the data
+
+# Freya will do relabeling here:
+
+
+# Data splitting
+import torch, sklearn
+import data_reader
+import os
+import numpy as np
+
+# Open train.csv
```

Comment:
Task description said: "Create function which takes csv as input and splits it into train/test/val csv" - so it would be nice if this was wrapped in a function and saved the differet csvs🙏 Also, no magic numbers, so the ratio of train/validation should also be given